daq_macros.py

```diff
@@ -26,7 +26,10 @@
 from collections import OrderedDict
 from scans import (zebra_daq_prep, setup_zebra_vector_scan,
                    setup_zebra_vector_scan_for_raster,
-                   setup_vector_program)
+                   setup_vector_program, setup_eiger_exposure,
+                   setup_eiger_triggers, setup_eiger_arming,
+                   setup_eiger_stop_acquire_and_wiat
```

Comment:
wiat -> wait

Reply:
fixed in https://github.com/NSLS-II/lsdc/pull/101/commits/258e1befc4d7365a64d93d84bf193806ed91a7e0